tests/unit_tests/test_predict.py

```diff
@@ -0,0 +1,28 @@
+import os
+import unittest
+import tempfile
+
+from super_gradients.common.object_names import Models
+from super_gradients.training import models
+
+
+class TestModelPredict(unittest.TestCase):
+    def setUp(self) -> None:
+        rootdir = os.path.dirname(os.path.dirname(os.path.dirname(__file__)))
+        self.images = [
+            os.path.join(rootdir, "documentation", "source", "images", "examples", "countryside.jpg"),
+            os.path.join(rootdir, "documentation", "source", "images", "examples", "street_busy.jpg"),
+            "https://cdn-attachments.timesofmalta.com/cc1eceadde40d2940bc5dd20692901371622153217-1301777007-4d978a6f-620x348.jpg",
```

Comment:
Let's ask @ranrubin to put some image on our S3 instead. 
We do not control this domain and test may break at any time.